src/internal/config.ts

```diff
@@ -16,6 +18,20 @@ export const config = {
    */
   onUnhandledError: null as ((err: any) => void) | null,
 
+  /**
```

Comment:
Nice doc comment!

Reply:
LMAO... thanks GitHub.. Made this look sarcastic. I really did like the comment below this line.